tensorflow/c/env.h

```diff
@@ -152,6 +152,11 @@ TF_CAPI_EXPORT extern TF_StringStream* TF_GetChildren(const char* filename,
 // The caller is responsible for freeing the list (see TF_StringStreamDone).
 TF_CAPI_EXPORT extern TF_StringStream* TF_GetLocalTempDirectories(void);
 
+// Creates a temporary file name with an extension.
+// The caller is responsible for freeing the returned pointer.
+TF_CAPI_EXPORT extern void TF_GetTempFileName(const char* extension,
```

Comment:
Actually, let's make this return the name of the tempfile instead of using an out parameter.

Reply:
Done